docs/customizing.rst

```diff
@@ -0,0 +1,231 @@
+..
+    This file is part of Invenio.
+    Copyright (C) 2020 CERN.
+
+    Invenio is free software; you can redistribute it and/or modify it
+    under the terms of the MIT License; see LICENSE file for more details.
+
+Customizing
+===========
+
```

Comment:
I think quick introductions are always good, to have a fast view of whats in there. For example:

In section one we cover this and in section two this this and this

Mainly to give the user the power to choose were to go, being informed.